sdk/nodejs/eventhub/zMixins.ts

```diff
@@ -228,7 +228,7 @@ export class TopicEventSubscription extends appservice.EventSubscription<TopicCo
     }
 }
 
-interface EventHubBindingDefinition extends appservice.BindingDefinition {
+export interface EventHubBindingDefinition extends appservice.BindingDefinition {

```

Comment:
Consider marking this `internal`, we probably don't want end users to see this type

Reply:
Does this d1e6a93be7c9f1a6d23aaf3d606c58d299156a35 do the trick?

Reply:
It should, yes!

Reply:
could you also put on a note on it sayin that it is exported for use by the iot module?

Reply:
Done in 4befc8cc16a7a663e608029b69f0fd4a8d66b1e2